docs/development.md

```diff
@@ -0,0 +1,7 @@
+# Development
+
+{{ include_file('CONTRIBUTING.md') }}
+
+{{ include_file('CODE_OF_CONDUCT.md') }}
+
+{{ include_file('contributors.md') }}
```

Comment:
Would be nice if these were added to the development page's TOC.

Reply:
Fixed!